src/redist/targets/GenerateBundledVersions.targets

```diff
@@ -360,6 +360,19 @@
           "
           />
 
+      <NativeAOTRuntimePackRids Include="
+          @(Net80NativeAOTRuntimePackRids);
+          linux-x64;
+          linux-arm64;
+          linux-musl-x64;
+          linux-musl-arm64;
+          win-x64;
+          win-arm64;
+          browser-wasm;
+          wasi-wasm;
```

Comment:
I assume that these two could be a bit controversial. There are no official runtime packs for these two RIDs and they are only relevant for the NativeAOT/LLVM experiment in the dotnet/runtimelab.

However, moving full to the runtime pack approach in .NET 9 would mean that NativeAOT/LLVM has to adapt. It's non-trivial to update the supported RID list from MSBuild in an efficient way since it requires to be done early in the restore process (ie. one cannot make a NuGet that adds `browser-wasm` to the list).

When doing NativeAOT publish with a RID not on this list the runtime pack is silently not downloaded and resolved, resulting in a build with no runtime libraries and bogus errors later on. There's no explicit check for this case and no user friendly error. On the other hand, with a RID on this list that has no corresponding NuGet package you get a fairly meaningful error about the NuGet package not being found.

cc @SingleAccretion

Reply:
> On the other hand, with a RID on this list that has no corresponding NuGet package you get a fairly meaningful error about the NuGet package not being found.

Yes I agree that is much better.